app/views/mailing_list/steps/_name.html.erb

```diff
@@ -1,9 +1,7 @@
-<h1>Get personalised information and updates about getting into teaching</h1>
+<h1>Get personalised advice about getting into teaching</h1>
```

Comment:
Advice has always been a difficult word to use in this context, as the TTA service is the more _advice_-y service. Any other potential options you can recommend? We could always keep either information or updates, but happy to look at others, maybe something like 'content'?

Reply:
Yea - had the same thought! I thought we could try getting away from 'updates' given the feedback that users don't really know what that means - and 'information' is not very juicy! You're right that 'advice' could come across as an oversell for what is essentially emails. 

I was thinking perhaps 'guidance'? Also not that juicy but at least it implies you're being supported, rather than 'information' which feels quite impersonal?